setup_users_BAD.py

```diff
@@ -0,0 +1,100 @@
+from nlp4all import db, bcrypt
+from nlp4all.models import User, Role, Organization, Project
+from nlp4all.models import TweetTagCategory, Tweet, User, Organization, Project, BayesianAnalysis
+from nlp4all import db, bcrypt
+import json, os
+from datetime import datetime
+import time
+import json
+
+db.drop_all()
+
+
+db.create_all()
+db.session.commit()
+
+admin_role = Role(name='Admin')
+db.session.add(admin_role)
+student_role = Role(name='Student')
+db.session.add(admin_role)
+teacher_role = Role(name='Teacher')
+db.session.add(admin_role)
+db.session.commit()
+
+hp = bcrypt.generate_password_hash("Hermes_2014")
```

Comment:
this is really really bad and we will have to address this and i will consider purging it from the entire repos history

Reply:
🤦‍♀️🤦‍♀️🤦‍♀️🤦‍♀️🤦‍♀️